### PR TITLE
chore(ui): Tailwind eslint config

### PR DESCRIPTION
**Pull request by jashanbhullar**
# Overview
Tailwind eslint config for consistent styling. 

## What I've done
- added config in .eslintrc.js 
- added package eslint-plugin-tailwindcss@3.17.4 
- automatic fix using `yarn format` 

## What I haven't done
NA 

## How I tested
NA

## Screenshot
NA

## Which point I want you to review particularly
The PR will be impossible so there will be very little manual changes. Check the individual commmits and not the file changes as a whole. 

## Memo
Review using individual commits 

## Summary by CodeRabbit

- **Style**
  - Improved visual consistency by updating and reordering CSS classes across multiple components, including `Carousel`, `Checkbox`, `Command`, `ContextMenu`, `DataTable`, `Dialog`, `DropdownMenu`, `FileTree`, `Loading`, `NavigationMenu`, `Resizable`, `ScrollArea`, `Select`, `BaseButton`, and `IconButton`.
  - Enhanced layout and styling in the Editor and Dashboard features for elements such as panels, nodes, action bars, infobars, and toolboxes.
  - Adopted more generic class names for icon sizes, replacing specific height and width values with uniform `size` classes.

### CODE REVIEW

File: ui/src/components/Loading/index.tsx [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
         <div className="flex flex-col gap-5">
           <div className="flex gap-3">
             <FlowLogo
               id="loading-svg"
-              className="bg-red-900 bg-opacity-50 text-zinc-200 rounded-lg p-1 mb-8"
+              className="mb-8 rounded-lg bg-red-900 bg-opacity-50 p-1 text-zinc-200"
               style={{ height: "110px", width: "110px" }}
             />
-            <DoubleArrowRightIcon className="w-[110px] h-[110px] text-zinc-600" />
+            <DoubleArrowRightIcon className="size-[110px] text-zinc-600" />
```
[the remark this answers is not in the document]
Comment (jashanbhullar) [reply ui/src/components/Loading/index.tsx]: Going by what taildwind recommends

File: ui/src/features/Editor/components/Canvas/components/Nodes/GeneralNode/index.tsx [hunk 2]
```diff
@@ -52,15 +52,15 @@ const GeneralNode: React.FC<GeneralNodeProps> = ({ className, data, type, select
...
   return (
     <div
-      className="bg-zinc-800 rounded-sm"
+      className="rounded-sm bg-zinc-800"
       // onMouseEnter={() => setHovered(true)}
       // onMouseLeave={() => setHovered(false)}
       onDoubleClick={handleDoubleClick}>
       <div
-        className="flex relative w-[150px] z-[1001] rounded-sm bg-zinc-900/50 h-[25px]"
+        className="relative z-[1001] flex h-[25px] w-[150px] rounded-sm bg-zinc-900/50"
```
[the remark this answers is not in the document]
Comment (jashanbhullar) [reply ui/src/features/Editor/components/Canvas/components/Nodes/GeneralNode/index.tsx:60]: Thanks. This was helpful

File: ui/src/features/Editor/components/LeftPanel/components/TransformerList.tsx [hunk 3]
```diff
@@ -3,20 +3,20 @@ import { Lightning } from "@phosphor-icons/react";
...
-      <div className="flex gap-2 items-center">
-        <Lightning className="w-[15px] h-[15px]" weight="thin" />
+      <div className="flex items-center gap-2">
+        <Lightning className="size-[15px]" weight="thin" />
         <p className="text-sm font-extralight">Transformer</p>
       </div>
-      <div className="flex gap-2 items-center">
-        <Lightning className="w-[15px] h-[15px]" weight="thin" />
+      <div className="flex items-center gap-2">
+        <Lightning className="size-[15px]" weight="thin" />
```
[the remark this answers is not in the document]
Comment (jashanbhullar) [reply ui/src/features/Editor/components/LeftPanel/components/TransformerList.tsx:19]: It's okay for now. Only temporary code

File: ui/src/features/Editor/components/RightPanel/index.tsx [hunk 4]
```diff
@@ -28,30 +28,30 @@ const RightPanel: React.FC<Props> = ({ selected }) => {
...
-        className="flex bg-zinc-800 border-l border-zinc-700 fixed right-0 h-full w-[350px] transition-all"
+        className="fixed right-0 flex h-full w-[350px] border-l border-zinc-700 bg-zinc-800 transition-all"
         style={{
           transform: `translateX(${selected ? "0" : "100%"})`,
           transitionDuration: selected ? "500ms" : "300ms",
           transitionProperty: "transform",
           transitionTimingFunction: "cubic-bezier(0.4, 0, 0.2, 1)",
         }}>
-        <div className="bg-zinc-900/50 w-full h-full py-4 pl-4 pr-2">
+        <div className="size-full bg-zinc-900/50 py-4 pl-4 pr-2">
```
[the remark this answers is not in the document]
Comment (jashanbhullar) [reply ui/src/features/Editor/components/RightPanel/index.tsx:54]: Not changing code for now

**Review by KaWaite — APPROVED** [review]

**Pull request merged**